projects/packages/my-jetpack/_inc/components/product-detail-card/style.module.scss

```diff
@@ -1,26 +1,37 @@
 .card {
-	background-color: var(--jp-white);
+	background-color: var( --jp-white );
 	height: 100%;
+	padding: calc( var( --spacing-base ) * 6 );
+	position: relative;
+
+	.container {
+		height: 100%;
+		display: flex;
+		flex-direction: column;
+	}
 }
 
 .card-header {
 	display: flex;
 	align-items: center;
-	background: var(--jp-black);
-	color:var(--jp-white);
-	border-radius: var(--jp-border-radius) var(--jp-border-radius) 0 0;
-	padding: 0 var(--spacing-base); // 8px
-	height: 32px;
+	background: var( --jp-black );
+	color: var( --jp-white );
+	border-radius: var( --jp-border-radius ) var( --jp-border-radius ) 0 0;
+	padding: 0 var( --spacing-base );
+	height: calc( var( --spacing-base ) * 4 );
+	position: absolute;
+	top: 0;
+	left: 0;
```

Comment:
By default it's not `top:0` and `left:0`?

Reply:
it doesn't locate the header properly when removing these values

Reply:
CSS magic hehe